test/unit/general/contextBrokerOAuthSecurityAccess-test.js

```diff
@@ -0,0 +1,256 @@
+/*
```

Comment:
I haven't review this in detail, as I'm not an Oauth expert... but maybe somebody should do...

NTC (just informative)